.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
py.test creates `.cache` to rerun failed tests. You could add it to your global `.gitignore` so it doesn't get committed